src/lib.rs

```diff
@@ -57,6 +57,7 @@ impl SerialPort {
     ///
     /// This function is unsafe because the caller must ensure that the given base address
     /// really points to a serial port device.
+    #[cfg(not(feature = "stable"))]
```

Comment:
Why not use `#[cfg(feature = "nightly")]` here? Const functions are a nightly feature, so it makes sense to make the function const only when the feature is enabled.

Reply:
changed